tests/test_rotation.py

```diff
@@ -0,0 +1,47 @@
+from unittest import TestCase
+import numpy as np
+
+from scipy.spatial.transform import Rotation as sp_rot
+from aspire.utils.rotation import Rotation
+from aspire.utils.types import utest_tolerance
+
+
+class UtilsTestCase(TestCase):
+    def setUp(self):
```

Comment:
You don't need 3 attributes here - just have `self.rot_obj` and look inside it to get what you want.

Reply:
Removed in commit 707aace.